apps/package.json

```diff
@@ -134,7 +134,7 @@
     "karma-mocha-reporter": "^2.2.3",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack-with-fast-source-maps": "^1.10.0",
+    "karma-webpack": "^2.0.9",
```

Comment:
Needed to fix `webpack configuration misses the property 'entry'`, see https://github.com/webpack/webpack/issues/3909#issuecomment-271952137.

The [`karma-webpack-with-fast-source-maps` fork](https://github.com/aaronjensen/karma-webpack) hasn't ported over this fix yet.